src/project/WorkingSetSort.js

```diff
@@ -219,6 +220,7 @@ define(function (require, exports, module) {
 
     /**
      * The Command ID
+     * @private
```

Comment:
Sort object is returned from `register` API and hence this has to be treated as public. So we need to retain all the APIs in Sort.